azext_iot/central/_help.py

```diff
@@ -136,6 +136,51 @@ def _load_central_devices_help():
                 az iot central app device registration-summary --app-id {appid}
         """
 
+    helps[
+        "iot central app device run-command"
+    ] = """
+            type: command
+            short-summary: Run a command on a device and view associated response. Does NOT monitor property updates that the command may perform.
```

Comment:
Isn't the second sentence assumed? This isn't an active monitoring command.

Reply:
We have a concept of async command that we should chat about offline